src/sql/mod.rs

```diff
@@ -25,8 +25,8 @@ pub(crate) enum Part {
 }
 
 impl SqlBuilder {
-    pub(crate) fn new(template: &str) -> Self {
-        let mut iter = template.split('?');
+    pub(crate) fn new(template: &str, arg: &str) -> Self {
+        let mut iter = template.split(arg);
```

Comment:
aside, rather surprised this doesn't at least have usual escape hatch like `\?` or `??` which this change would somewhat complicate fixing